docs/user/managing-projects-feeds.md

```diff
@@ -52,10 +52,37 @@ Feed Versions are created from the main Feed Source profile page. There are thre
 
 3. **Import From the GTFS Editor**: Select "From snapshot" from the `+ Create new version` dropdown. The list of snapshots should now be visible showing any available snapshots of the feed in the Editor. Select the desired snapshot by clicking the "Publish" button to publish the snapshot as a new version.
 
+4. **Service Period Merge** (certain Data Tools configurations only): If a Feed Source has two or more Feed Versions, a new Feed Version can be created by merging two versions representing a transit agency's service over different time periods. While viewing a particular Feed Version, click **Merge with version** (underneath the map view) to select which past version you would like to merge with.
```

Comment:
"Merge with version": this command should be outlined with backquote (\`) for consistency with the rest of the document.